### PR TITLE
TW-142 Adds KB on WF visibility

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

Adds a KB doc on different ways to check on WF visibility.

## Notes to reviewers

[Preview](https://temporal-documentation-cw5ar1il1.preview.thundergun.io/kb/visibility-status-of-running-workflow)

### CODE REVIEW

File: kb/visibility-status-of-running-workflow.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+tags:
+  - kb-article
+date: 2022-11-7T00:00:00Z
+---
+
+To check how many Workflows are currently running, you must set up the [Advanced Visibility](/concepts/what-is-advanced-visibility) feature, which depends on an integration with Elasticsearch.
+
+<!-- truncate -->
+
+Once you've integrated Elasticsearch with your Temporal Cluster, you can get information into the visibility of your running Workflows.
```
Comment (djmagee) [inline kb/visibility-status-of-running-workflow.md:13]:
```suggestion
After you integrated Elasticsearch with your Temporal Cluster, you can get information into the visibility of your running Workflows.
```

File: kb/visibility-status-of-running-workflow.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+For more examples, see [List Filter examples](/visibility#example-list-filters).
+
+## Using APIs
+
+Alternatively, you can use the following APIs with your Visibility Query.
+
+- [ListWorkflowExecutions](https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/service.proto#L279)
+- [CountWorkflowExecutions](https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/service.proto#L291)
+
+It isn’t recommended using the APIs with high-rate calls.
```
Comment (djmagee) [inline kb/visibility-status-of-running-workflow.md:46]:
```suggestion
We don’t recommend using the APIs with high-rate calls.
```

File: kb/visibility-status-of-running-workflow.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+For more information, see [Search Attributes](/concepts/what-is-a-search-attribute/).
+
+:::note
+
+All APIs lists are paginated.
```
Comment (djmagee) [inline kb/visibility-status-of-running-workflow.md:52]:
```suggestion
All lists from APIs are paginated.
```

File: kb/visibility-status-of-running-workflow.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+tags:
+  - kb-article
+date: 2022-11-7T00:00:00Z
+---
+
+To check how many Workflows are currently running, you must set up the [Advanced Visibility](/concepts/what-is-advanced-visibility) feature, which depends on an integration with Elasticsearch.
+
+<!-- truncate -->
+
+After you integrated Elasticsearch with your Temporal Cluster, you can get information into the visibility of your running Workflows.
```
Comment (djmagee) [inline kb/visibility-status-of-running-workflow.md:13]:
```suggestion
After you integrate Elasticsearch with your Temporal Cluster, you can get information into the visibility of your running Workflows.
```
Oops. I missed this earlier. My bad.

**Review by djmagee — APPROVED** [review]
I missed one edit during my earlier pass.

**Review by flossypurse — CHANGES_REQUESTED** [review]
Just making sure this doesn't get merged too soon.
Per my last chat with @rachfop , we are looking at whether this should be massaged into concept explanations

**Pull request closed without merging**